staging/com.smartnews/dataset.tt

```diff
@@ -0,0 +1,90 @@
+dataset @com.smartnews language "en" {
+    query := @com.smartnews.top_articles()
+    #_[utterances=["smartnews",
+                   "smart news",
+                   "smartnews top",
+                   "smart news top",
+                   "news",
+                   "top news",
+                   "top smartnews",
+                   "top smart news",
+                   "news from smartnews",
+                   "news on smartnews",
+                   "breaking news",
+                   "smart news",
+                   "today news",
+                   "popular news",
+                   "important news",
+                   "today's news",
+                   "global news",
+                   "hot topic",
+                   "hot news",
+                   "high rank news"]]
+    #[id=61614268]
+    #[name="TopArticles"];
+
+    query (p_counter :Number) := @com.smartnews.top_articles(counter=p_counter)
```

Comment:
It would be preferrable, for compatibility with the templates, to use a limit expression rather than input parameter for this:
```
query (p_counter :Number) := @com.smartnews.top_articles()[1 : p_counter]
```